### PR TITLE
Task a Tiny JS World

### PR DESCRIPTION
**Pull request by OlexiyDobroskok**
### A Tiny JS World

[Demo](https://olexiydobroskok.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/OlexiyDobroskok/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by OlexiyDobroskok** [comment]
Made some self-checks to fix most common issues

File: submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+];
+
+const residentsProperties = aTinyWorld.map((resident) => {
+  const allProperties = [];
+  properties.forEach((prop) => {
+    if (resident[prop] !== 0) {
+      allProperties.push(resident[prop]);
+    }
+  });
+  return allProperties;
```
Comment (OleksiyRudenko) [inline submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js:69]: A classic use case for `Array#map`

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@OlexiyDobroskok quite a decent job done!
Let's polish it.

File: submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js (hunk 1 above)
Comment (OlexiyDobroskok) [reply submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js]: Done =)

**Comment by OlexiyDobroskok** [comment]
Made the code a little cleaner :)

File: submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "hands",
+  "saying",
+  "friends",
+];
+
+const residentsProperties = aTinyWorld.map((resident) => {
+  return properties
+    .map((prop) => {
+      return resident[prop];
+    })
```
Comment (OleksiyRudenko) [inline submissions/OlexiyDobroskok/task_aTinyJsWorld/index.js:77]: If an arrow function has a single statement and the statement is `return`, you do not need neither surrounding curly braces nor `return` keyword.
Also if an arrow function has exactly one argument parentheses can be skipped.

Compare readability:
```
.map((prop) => {
  return resident[prop];
})

.map(prop => resident[prop])
```

Same refers to the enclosing function in this case. Proper formatting will help to keep nested code radable.

**Review by OleksiyRudenko — APPROVED** [review]
@OlexiyDobroskok great job!
CHeck the comment below for a learning to use.

**Pull request merged**